src/main/java/teammates/common/datatransfer/FeedbackParticipantType.java

```diff
@@ -17,6 +21,28 @@
     // Used by feedbackResponseComment:
     GIVER(false, false, true, "", "", "");
 
+    public static final List<FeedbackParticipantType> GIVERS;
+    static {
+        List<FeedbackParticipantType> giverInitializer = new ArrayList<FeedbackParticipantType>();
+        for (FeedbackParticipantType participantType : FeedbackParticipantType.values()) {
+            if (participantType.isValidGiver()) {
+                giverInitializer.add(participantType);
+            }
+        }
+        GIVERS = Collections.unmodifiableList(giverInitializer);
```

Comment:
It is used to directly generate the `<option>` for feedback path:
https://github.com/TEAMMATES/teammates/blob/fd9dfb48da8508bbc7fdf8058d3f9ac14f9114e7/src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionFeedbackPathSettings.tag#L64